src/service/public-key.js

```diff
@@ -83,6 +83,14 @@ class PublicKey {
         util.throw(400, 'Provided email address does not match a valid user ID of the key');
       }
     }
+
+    const last = array => array[array.length - 1];
+    const domainNames = key.userIds.map(userId => last(userId.split('@')));
+    // if all email addresses are not in the domain, an error will be thrown
+    if (!domainNames.some(domainName => config.publicKey.domainNames.includes(domainName))) {
```

Comment:
すべてのメールアドレスにおいて, ドメイン名が正しいかどうか確認するのであればこのような気がする
違ってたら申し訳ない
```suggestion
    if (!domainNames.every(domainName => config.publicKey.domainNames.includes(domainName))) {
```

Reply:
`approvers.dev` なメールアドレスが User ID に含まれていればよくて，他のドメイン名の User ID を持っていても構わないので（ユースケースとして，普段使っている主鍵，私なら `root@siketyan.dev` に `siketyan@approvers.dev` を追加することを想定しているので）

Reply:
理解しました

Reply:
であればコメントが違うな

Reply:
ヒッ

Reply:
if _all_ email addresses are **NOT** in the domain じゃだめなのか

Reply:
if no email addresses are in the domain のほうがいいか

Reply:
ごめん，どっちとも取れる英語だ
`If no uid email address is in `approvers.dev`,` とか?

Reply:
@JJ1LFC 00f00b0